pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "metricflow"
-version = "0.93.0"
+version = "0.100"
```

As @WilliamDee pointed out this should be:
```suggestion
version = "0.100.0"
```